Documentation/dev-guide/apispec/swagger/etcdserver/etcdserverpb/rpc.swagger.json

```diff
@@ -0,0 +1,1239 @@
+{
```

Comment:
this is still not good location. can we just put it under /apispec/swagger instead of apispec/swagger/etcdserver/etcdserverpb/